spec/diddocuments.md

```diff
@@ -184,8 +185,100 @@ Secp256k1 public keys must be converted to a verification method with a type of
 
 TODO: Add example of Secp256k1 key in a KEL and the resultant verification method
 
+##### Thresholds
+If the current signing keys threshold (the value of the `kt` field) is greater than 1, then in addition to the verification
+methods generated according to the rules in the previous sections, another verification method
+with a type of `ConditionalProof2022` will be generated in the DID document. This verification method type is defined here:
+https://w3c-ccg.github.io/verifiable-conditions/ (TODO: add proper references).
+
+The `id` property of the verification method MUST be a relative DID URL and use the AID as the value of the fragment component, e.g., `"id": "#<aid>"`.
```

Comment:
Why is this?  Do the other keys get created in the KERI event stream for this DID, or are are the identifiers created in their on KERI event stream?  It makes sense if they are in the same KERI event stream for these to be relative references (they are defined/managed within by this DID).

Reply:
Resolved in the call.  When a multi-sig identifier is built, each controller specifies their (literal) public keys in the events that create/evolve the identifier. They do not specify _references_ to their keys -- e.g. they don't specify a DID that is resolved to retrieve their key.  As such, each of the public keys are always put directly into the DIDDoc (as indicated in the example), and a relative reference within the DIDDoc is used in the ConditionalProof2022 list.

Each of the controllers manage their own key pairs as they want -- for example using KERI -- but where and how those keys are managed is not relevant in the multi-sig `did:webs` identifer.

Reply:
Agreed.